### PR TITLE
Fix default value, add doc, fix norm #37

### LINKED ISSUE #37: Add values for boiler and correct norm
**Describe the bug**
1. `dTTra_nominal` is missing a default assumption in the GasBoiler model.
2. EN 15251 is outdated

### PR DESCRIPTION
**Pull request by FWuellhorst**
Closes #37 

@MZuschlag : Could you review this one? I found the EN 303-1 to give the value of 10-20 K as default.
Also, i fixed a wrong norm reference and added docs to DHW.

### CODE REVIEW

**Review by FWuellhorst — COMMENTED** [review]
@MZuschlag : I reverted the file, can you do your review? :)

File: BESMod/Systems/Hydraulical/Generation/GasBoiler.mo [hunk 1]
```diff
@@ -146,4 +148,9 @@ equation
       index=1,
       extent={{-6,3},{-6,3}},
       horizontalAlignment=TextAlignment.Right));
+  annotation (Documentation(info="<html>
+<h4>Bottom-up parameters</h4>
+<p><br>The value for dTTra_nominal is based on EN 303-1:2017, which requires a temperature spread between 10 and 20 K. </p>
```
Comment (MZuschlag) [inline BESMod/Systems/Hydraulical/Generation/GasBoiler.mo:153]: You can check e.g. Verordnung (EU) Nr. 813/2013 additionally

File: BESMod/Systems/Demand/Building/BuildingsRoomCase600FF.mo [hunk 2]
```diff
@@ -9,7 +9,7 @@ model BuildingsRoomCase600FF
     AZone={roo.AFlo},
     nZones=1);
   parameter Modelica.Units.SI.TemperatureDifference dTComfort=2
-    "Temperature difference to room set temperature at which the comfort is still acceptable. In DIN EN 15251, all temperatures below 22 °C - 2 K count as discomfort. Hence the default value. If your room set temperature is lower, consider using smaller values.";
+    "Temperature difference to room set temperature at which the comfort is still acceptable. In DIN EN 16798-1, all temperatures below 22 °C - 2 K count as discomfort. Hence the default value. If your room set temperature is lower, consider using smaller values.";
```
Comment (MZuschlag) [inline BESMod/Systems/Demand/Building/BuildingsRoomCase600FF.mo:12]: Further down you use the international standard. Maybe you can refer to EN 16798-1:2019

File: BESMod/Systems/Demand/Building/TEASERThermalZone.mo [hunk 3]
```diff
@@ -16,7 +16,7 @@ model TEASERThermalZone
 
   parameter Boolean use_verboseEnergyBalance=true   "=false to disable the integration of the verbose energy balance";
   parameter Modelica.Units.SI.TemperatureDifference dTComfort=2
-    "Temperature difference to room set temperature at which the comfort is still acceptable. In DIN EN 15251, all temperatures below 22 °C - 2 K count as discomfort. Hence the default value. If your room set temperature is lower, consider using smaller values.";
+    "Temperature difference to room set temperature at which the comfort is still acceptable. In DIN EN 16798-1, all temperatures below 22 °C - 2 K count as discomfort. Hence the default value. If your room set temperature is lower, consider using smaller values.";
```
Comment (MZuschlag) [inline BESMod/Systems/Demand/Building/TEASERThermalZone.mo:19]: Further down you use the international standard. Maybe you can refer to EN 16798-1:2019

File: BESMod/Systems/Hydraulical/Generation/GasBoiler.mo (hunk 1 above)
Comment (FWuellhorst) [reply BESMod/Systems/Hydraulical/Generation/GasBoiler.mo:153]: Thanks, but this does not add information :)

**Review by MZuschlag — APPROVED** [review]

**Pull request merged**